src/content/configuration/module.mdx

```diff
@@ -49,6 +49,27 @@ module.exports = {
 };
 ```
 
+## `module.unsafeCache`
+
+`boolean` `function ()`
```

Comment:
I think this can be `function (module)` since `module.unsafeCache` becomes the cache predicate if it's set:
https://github.com/webpack/webpack/blob/e607681/lib/NormalModuleFactory.js#L163-L166

which in turn gets passed a copy of the module: 
https://github.com/webpack/webpack/blob/e607681/lib/NormalModuleFactory.js#L561

Reply:
Yes good point! 
```suggestion
`boolean` `function (module)`
```